### PR TITLE
[Doc] Update README

### PR DESCRIPTION
**Pull request by jpeng-ms**
## Purpose
This PR is meant to start a discussion on how we handling version number in the pod file. I believe we should encourage Contoso to always use latest version (therefore, I am proposing to remove '1.0.0' from podfile example). But I wanted to hear what others might think just in case I missed anything.

Sister PR in MSDoc: https://github.com/MicrosoftDocs/azure-docs-pr/pull/210316/files

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
N/A
## What to Check
Verify that the following are valid
* ...
N/A
## Other Information

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Get started with Azure Communication Services by using the UI Library to integra
 CocoaPods is a dependency manager for Cocoa projects. To set up with CocoaPods visit their [Getting Started Guide](https://guides.cocoapods.org/using/getting-started.html). To integrate UI Mobile Library into your Xcode project using CocoaPods, specify it in your `Podfile`:
 
 ```ruby
-pod 'AzureCommunicationUICalling', '1.0.0'
+pod 'AzureCommunicationUICalling'
```
Comment (JoshuaLai) [inline README.md:35]: Why do we want to remove the pinning here? 
I think ideally we want to write a script to update all these tiny little version numbers

Comment (jpeng-ms) [reply README.md:35]: right, I was thinking to remove it so we don't need to update them anymore

Comment (jpeng-ms) [reply README.md:35]: and here: https://github.com/MicrosoftDocs/azure-docs-pr/blob/242e5ccfe2391eae7d65fe9eacbb0c87cee8122b/articles/communication-services/quickstarts/ui-library/includes/get-started-call/ios.md?plain=1#L46

**Comment by jpeng-ms** [comment]
I am keeping whatever we have today for now, maybe in the future we could have a script to automate this process.

File: README.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ Get started with Azure Communication Services by using the UI Library to integra
 CocoaPods is a dependency manager for Cocoa projects. To set up with CocoaPods visit their [Getting Started Guide](https://guides.cocoapods.org/using/getting-started.html). To integrate UI Mobile Library into your Xcode project using CocoaPods, specify it in your `Podfile`:
 
 ```ruby
-pod 'AzureCommunicationUICalling', '1.0.0'
+pod 'AzureCommunicationUICalling, 1.1.0-beta.1'
```
Comment (jpeng-ms) [inline README.md:35]: this matches to https://github.com/Azure/communication-ui-library-ios/pull/338/files#diff-f0f2db3caab0941e5d538b6999827b39c98f491d933e37317733fe03dc5926d4R3

File: README.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ Get started with Azure Communication Services by using the UI Library to integra
 CocoaPods is a dependency manager for Cocoa projects. To set up with CocoaPods visit their [Getting Started Guide](https://guides.cocoapods.org/using/getting-started.html). To integrate UI Mobile Library into your Xcode project using CocoaPods, specify it in your `Podfile`:
 
 ```ruby
-pod 'AzureCommunicationUICalling', '1.0.0'
+pod 'AzureCommunicationUICalling', '1.1.0-beta.1'
```
Comment (jpeng-ms) [inline README.md:35]: this matches to https://github.com/Azure/communication-ui-library-ios/pull/338/files#diff-f0f2db3caab0941e5d538b6999827b39c98f491d933e37317733fe03dc5926d4R3

**Review by JoshuaLai — APPROVED** [review]

**Review by palatter — APPROVED** [review]

**Pull request merged**